lib/sneakers/worker.rb

```diff
@@ -61,11 +61,11 @@ def do_work(delivery_info, metadata, msg, handler)
           end
         rescue Timeout::Error
           res = :timeout
-          worker_error('timeout')
+          worker_error('timeout',msg)
         rescue => ex
           res = :error
           error = ex
-          worker_error('unexpected error', ex)
+          worker_error('unexpected error', msg, ex)
```

Comment:
But would the exception include the message?  If so then the application using Sneakers can handle the exceptions and do what it wishes or use Airbrake,et.al.